docs/deployment-guides/deploy-locally.md

```diff
@@ -99,7 +99,7 @@ In order to run the Azure Communication Services Authentication Server sample,
    > Note: You can also run the selected project using the Visual Studio.
 
 3. We have two ways of testing the backend service
-   - Using a manually generated token and calling the server directly
+   - Calling the server directly with an Azure Active Directory Access Token
```

Comment:
We should also changed these phrases to match the headings below, that will be more understandable